templates/landscape/install.html

```diff
@@ -715,6 +522,14 @@ <h3 class="p-heading--2">Install</h3>
             </div>
             <div class="p-strip u-no-padding--top">
               <ol class="p-list--divided">
+                <li class="p-list__item">
+                  <p class="u-no-padding--top">Define parameters in variables</p>
+                  <p>Self-hosted Landscape users should set LANDSCAPE_ACCOUNT_NAME as <code>standalone</code>, Landscape SaaS customers should specify their <u>account name</u> set LANDSCAPE_FQDN as <code>landscape.canonical.com</code>.
```

Comment:
`<u>account name</u>` should really be a hyperlink to: https://landscape.canonical.com/settings